test/e2e/create_test.go

```diff
@@ -56,11 +56,10 @@ func creationTestSuite(t *testing.T) {
 	t.Run("NodeIP Arg", testNodeIPArg)
 	t.Run("NodeTaint validation", testNodeTaint)
 	t.Run("CSR Validation", func(t *testing.T) { testCSRApproval(t) })
-	t.Run("UserData validation", testUserData)
-	t.Run("UserData idempotent check", testUserDataTamper)
 	t.Run("Certificates", testCertificates)
 	t.Run("Node Logs", testNodeLogs)
 	t.Run("Metrics validation", testMetrics)
+	t.Run("UserData validation", testUserData)
```

Comment:
Is there a reason for changing the order of the tests?

Reply:
I put this at the end because it is now causing the deletion and creation of a node, so it didn't make sense for me to run some of the creation tests against the original node and some against the new node.

Reply:
Please call that out in the commit message.